src/dawn/Optimizer/StencilInstantiation.h

```diff
@@ -563,6 +566,13 @@ class StencilInstantiation : NonCopyable {
     return BoundaryConditionToExtentsMap_.find(stmt)->second;
   }
 
+  Array3i getFieldIDFromInitializedDimensionsMap(int FieldID) {
+    if(fieldIDToInitializedDimensionsMap_.count(FieldID) == 0) {
+      return Array3i{{1, 1, 1}};
```

Comment:
can we assert, [question mark]
I think Array{1,1,1} has not the same semantic as "I couldnt find the field"

Reply:
no we can't. This way temporaries would not work anymore. If the field is user-specified, it is found, otherwise we allow all accesses